mlir/include/mlir/Dialect/Rock/IR/MfmaInsnGroup.h

```diff
@@ -0,0 +1,124 @@
+//===- MfmaInsnGroup.h - MLIR to C++ for Rock conversion
+//---------------===//
+//
+// Part of the MLIR Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// This file implements code selection logic for Mfma instructions.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_MFMA_INSN_GROUP_H
+#define MLIR_MFMA_INSN_GROUP_H
+
+#include <set>
```

Comment:
Any reason why not using `llvm::DenseSet` or (if possible) `llvm::SmallDenseSet`?

Reply:
I think this is a left over and I never ended using std::set at all. Will get rid of this include.